core/Array.carp

```diff
@@ -166,13 +166,28 @@
         (aset-uninitialized! &na i (f (nth a i))))
       na))
 
+  (doc sum-count "Returns the sum of counts from an Array of Arrays.")
+  (defn sum-count [xs]
```

Comment:
This could also be expressed in a more succinct way using `copy-map` and `Array.sum`.

Reply:
@eriksvedang I wanted to avoid the cost of the intermediate allocation of an array for the results